src/docs/guides/14-how-to-set-up-analytics-for-ftw/index.md

```diff
@@ -18,6 +18,11 @@ tracker ID to the environment variable `REACT_APP_GOOGLE_ANALYTICS_ID`;
 
 That's it! You're good to go.
 
+> Note: Google Analytics doesn't work in hot-loading environment!
+>       <br/>
```

Comment:
Did this HTML element come from the GitHub editor? It is rendered as HTML, but looks a bit weird in the Markdown source. Maybe just use an extra empty line there?

Reply:
```suggestion
>
```